content/en/docs/refguide/modeling/integration/published-odata-services/odata-operations.md

```diff
@@ -1,28 +1,28 @@
 ---
-title: "OData Query Options"
-url: /refguide/odata-query-options/
+title: "OData Operations"
+url: /refguide/odata-operations/
```

Comment:
There is a procedure for moving pages to prevent 404 for existing links.

Reply:
At second thought, "OData operations" can also be confusing, as it is also the umbrella term for OData actions and functions. "Working with OData", maybe?

Reply:
I thing "OData Query Options" is too bad. It's about the options that clients have when querying this OData service.